src/js/views/Project/Dependencies.jsx

```diff
@@ -7,6 +7,7 @@ import { httpDelete, httpGet, httpPost, lookupNamespaceByID } from '../../utils'
 import { useTranslation } from 'react-i18next'
 import { ModalForm } from '../../components/Form/ModalForm'
 import { jsonSchema } from '../../schema/ProjectDependencies'
+import { AutomationList } from './Create/AutomationList'
```

Comment:
Should we move _AutomationList.jsx_ out of _Project/Create_ and into _Project_ at this point?

Reply:
Yeah probably a good call.